### PR TITLE
DMP-4225: Flyway script to run automated tasks as different users

### PR DESCRIPTION
**Pull request by Ben-Edwards-cgi**
### Change description ###
In order to more easily identify the automated task that changed data using the "created_by" and "last_modified_by" columns we would like each automated task to run as a different user.

Create a flyway script to add the required number of users, giving hard-coded negative IDs.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by jackmaloney — APPROVED** [review]

File: src/main/resources/db/migration/common/V1_403__automated_task_users.sql [hunk 1]
```diff
@@ -0,0 +1,94 @@
+INSERT INTO user_account
+VALUES (-6, NULL, 'System user for ProcessDailyList Job', NULL,
+        'systemProcessDailyListAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true,
```
Comment (mestebanez) [inline src/main/resources/db/migration/common/V1_403__automated_task_users.sql:3]: Dont know why, but see 31 tasks but 28 in the code base

File: src/main/resources/db/migration/common/V1_403__automated_task_users.sql [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        true, 'systemProcessDailyListAutomatedTask'),
+       (-7, NULL, '', NULL,
+        'systemCloseOldUnfinishedTranscriptionsAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true,
```
Comment (davet1985) [inline src/main/resources/db/migration/common/V1_403__automated_task_users.sql:6]: Any specific reason to prefix them all with `system`? Seems overkill as they are set as "system" users as well...

File: src/main/resources/db/migration/common/V1_403__automated_task_users.sql (hunk 1 above)
Comment (jackmaloney) [reply src/main/resources/db/migration/common/V1_403__automated_task_users.sql:3]: There were 3 new ones added to Flyway last week, so this is expected.

File: src/main/resources/db/migration/common/V1_403__automated_task_users.sql (hunk 2 above)
Comment (jackmaloney) [reply src/main/resources/db/migration/common/V1_403__automated_task_users.sql:6]: This was to follow the pattern with the other system users already in the database.

**Review by davet1985 — APPROVED** [review]

**Pull request merged**